apps/aecore/src/aec_capabilities.erl

```diff
@@ -0,0 +1,90 @@
+-module(aec_capabilities).
+
+-behavior(gen_server).
+
+
+-export([ set_capability/2
+        , get_capability/1
+        , delete_capability/1
+        , get_capabilities/0
+        , register_peer/2
+        , remove_peer/1
+        , peers_with_capability/1 ]).
+
+%% Gen_server stuff
+-export([ start_link/0 ]).
+
+-export([ init/1
+        , handle_call/3
+        , handle_cast/2
+        , handle_info/2
+        , terminate/2
+        , code_change/3 ]).
+
+start_link() ->
+    gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
+
+get_capability(Key) ->
+    call({get_capability, Key}).
+
+set_capability(Key, Value) ->
+    call({set_capability, Key, Value}).
+
+delete_capability(Key) ->
+    call({delete_capability, Key}).
+
+get_capabilities() ->
+    call(get_capabilities).
+
+register_peer(Peer, Capabilities) when is_list(Capabilities) ->
```

Comment:
It will be used as a map, currently there is a sample capability which could be used for syncing

```
capabilities_template(<<"chain_poi">>, ?PING_VSN_2) ->
    [{chain_poi, #{ items => [ {height, int}
                             , {root_hash, binary}
                             , {genesis, binary}
                             , {top, binary}
                             , {poi, binary} ]}
     }];
```